yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -209,7 +209,8 @@ WHERE schemaname = ANY(ARRAY[string_to_array('$schema_list', '|')])
             read -r user_input
             if [[ "$user_input" != "y" && "$user_input" != "Y" ]]; then
                 echo "You can run ANALYZE manually on the affected schemas before retrying."
```

Comment:
Out of scope for this PR: but should we also report the tables involved, rather than just the schemas? 
is the concern that the output will have too many tables? 
cc: @shubham-yb; can't remember if you had brought it up or not.